README.md

```diff
@@ -105,35 +110,54 @@ With changes in Memgraph API, MAGE started to track version numbers. The table
 below lists the compatibility of MAGE with Memgraph versions.
 | MAGE version | Memgraph version |
 | ------------ | ----------------- |
+| >= 1.5.1 | >= 2.5.1 |
+| >= 1.5 | >= 2.5.0 |
 | >= 1.4 | >= 2.4.0 |
 | >= 1.0 | >= 2.0.0 |
 | ^0 | >= 1.4.0 <= 1.6.1 |
 
 ## How to install MAGE?
 
-There are two options to install MAGE. For the [Docker
+There are two options to install MAGE. 
+
+1) For the [Docker
 installation](#1-installing-mage-with-docker), you only need Docker installed.
-[To build from
+
+2) [To build from
 source](#2-installing-mage-locally-with-linux-package-of-memgraph), you will
-need **Python3**, **Make**, **CMake**, **Clang**, **UUID** and **Rust**.
+need to install the following on your machine:
```

Comment:
Can we avoid repeating this list twice in the document?

Reply:
Removed it from here, yes, you are right